packages/v3/src/esp-entity-table.ts

```diff
@@ -232,7 +241,10 @@ export class EntityTable extends LitElement implements RestAction {
   }
 
   _handleEntityRowClick(e: any) {
-    if (e?.currentTarget?.domain === "sensor") {
+    if (
+      e?.currentTarget?.domain === "sensor" ||
+      e?.currentTarget?.domain === "climate"
+    ) {
```

Comment:
Not important and not certain if this code is being transpiled, but if it is, a bit of redundant code can be reduced by doing something like `if (!e?.currentTarget) return;` as the first line within this function. That way we're not repeatedly checking and ensuring that `e` and `currentTarget` exists (if it does the first time, it will the next n times). Then also TS shouldn't require `?` for each reference (`e?.currentTarget?...` -> `e.currentTarget...`).

I know this wasn't you, but a good opportunity for tiny clean up if you're willing.